community-build/src/scala/dotty/communitybuild/projects.scala

```diff
@@ -357,8 +357,8 @@ object projects:
   lazy val stdLib213 = SbtCommunityProject(
     project       = "stdLib213",
     extraSbtArgs  = List("-Dscala.build.compileWithDotty=true"),
-    sbtTestCommand   = """library/compile""",
-    sbtPublishCommand = """set library/Compile/packageDoc/publishArtifact := false; library/publishLocal""",
+    sbtTestCommand   = """set Global / fatalWarnings := false; library/compile""",
+    sbtPublishCommand = """set Global / fatalWarnings := false; set library/Compile/packageDoc/publishArtifact := false; library/publishLocal""",
```

Comment:
community_build_c failed with `[error] bad option '-Xlint' was ignored`. I assume it's because this PR makes Scala 3 understand `-Werror` (before it only understood `-Xfatal-warnings`), so the `bad option` warning for `-Xlint` failed the build.